packages/flutter/lib/src/material/chip.dart

```diff
@@ -2795,3 +2811,83 @@ class _RenderChip extends RenderBox {
   @override
   bool hitTestSelf(Offset position) => deleteButtonRect.contains(position) || pressRect.contains(position);
 }
+
+class _LocationAwareInkRippleFactory extends InteractiveInkFeatureFactory {
+  const _LocationAwareInkRippleFactory(
+      this.hasDeleteButton, this.chipContext, this.deleteIconKey);
+
+  final bool hasDeleteButton;
+  final BuildContext chipContext;
+  final GlobalKey deleteIconKey;
+
+  @override
+  InteractiveInkFeature create(
+      {MaterialInkController controller,
+      RenderBox referenceBox,
```

Comment:
Update all these to use 2 spaced indent (the trailing comma will help with this)